capabilities/code-maintainability.md

```diff
@@ -5,28 +5,47 @@ In the lifespan of a typical line of code, it will likely be written once, edite
 As such, it makes sense to write code in a way that reduces the time required to comprehend it.
 Ideally, the software is built such that the reader can avoid needing to know how the whole system functions in order to make a local change.
 Furthermore, bugs are easier to spot when the surrounding code is easy to comprehend.
-The result of building code that's easy to maintain is more efficient delivery and better stability.
+The result of building code that's easy to maintain is more efficient delivery and better stability. But writing maintainable code requires an advanced set of skills — a set of skills that is developed by following the practices described in this section. 
 
-When difficult to maintain code is written, it can have a snowball effect.
-It requires an advanced set of skills to write maintainable code in a sea of hard to maintain code.
-Often, developers follow the path of least resistance by adding to the mess to get their new functionality added.
-There may be good reasons to follow such a path, however doing so further degrades the maintainability of the system.
-There is a delay between difficult to maintain code being introduced and the team experiencing a performance degradation.
-This delay makes it harder for non-technical team members to understand the consequences of implementing too many short-term solutions.
+<ed> Edit OK? I want to assure the reader that this "advanced set of skills" is indeed attainable and how. </ed>
```

Comment:
Can you remove the comments from the file itself? We don't want to include those. We can have our editorial conversation in the PR itself.

Reply:
To answer your question, how it's written feels repetitive. We're saying "set of skills" twice in a row. I suspect there's a way to get the point across without the repetition.

Reply:
Yes, will remove comments from the file and consolidate the language here.